app/workers/end_of_cycle/decline_by_default_worker.rb

```diff
@@ -0,0 +1,22 @@
+module EndOfCycle
+  class DeclineByDefaultWorker
+    include Sidekiq::Worker
+
+    def perform
+      declineable_applications.each do |application_form|
+        EndOfCycle::DeclineByDefaultService.new(application_form).call
```

Comment:
Have we considered passing the `ApplicationChoice` to this service rather than the Application Form? Are there any advantages to passing the ApplicationForm?

Reply:
This is a good question. I initially wrote it to just grab the application choice. but looking at similar jobs, we seem to have a preference for going via the application form. 

I can see a couple advantages in terms of scalability of the application form
- if we want to add some reporting along the lines of 'how many candidates did happen to', the application form is more useful. 
- If we want to add sending an email after rejection / decline, again, having a unique application form means we can do that more easily
- The job itself should be more efficient because we are iterating through a smaller number of records.

So I decided to maintain the existing pattern and use forms instead of choices, but I am not entirely convinced it's the best way. Happy to change it if you (or other reviewers) feel we should.